From e7186daf520d56f0d48fc4dfc8a3081f964b8f31 Mon Sep 17 00:00:00 2001
From: Tomas Popela <tpopela@redhat.com>
Date: Mon, 19 Sep 2016 10:27:15 +0200
Subject: [PATCH 393/595] Bug 771493 - Missing '>' in message reply

When parsing the HTML content to blocks we cannot look for the first "<br>"
token from the first position of input as when the input will start with the
same token, then we will later exit the processing loop prematurely as we won't
have a non-empty content to process.
---
 .../web-extension/e-editor-dom-functions.c         |  2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/webkit-editor/web-extension/e-editor-dom-functions.c b/modules/webkit-editor/web-extension/e-editor-dom-functions.c
index faa0648ae4..5abb9183d3 100644
--- a/modules/webkit-editor/web-extension/e-editor-dom-functions.c
+++ b/modules/webkit-editor/web-extension/e-editor-dom-functions.c
@@ -5169,7 +5169,7 @@ parse_html_into_blocks (EEditorPage *editor_page,
 	html = remove_new_lines_around_citations (input);
 
 	prev_token = html->str;
-	next_token = strstr (prev_token, "<br>");
+	next_token = strstr (prev_token + 1, "<br>");
 	processing_last = !next_token;
 
 	while (next_token || processing_last) {
-- 
2.15.0

